docs/mkdocs.yml

```diff
@@ -3,7 +3,7 @@ theme: readthedocs
 docs_dir: ${CMAKE_CURRENT_BINARY_DIR}
 site_url: http://nominatim.org
 repo_url: https://github.com/openstreetmap/Nominatim
-pages:
+nav:
```

Comment:
I know that newer versions of mkdocs are complaining but I'd like to leave that for a while to remain backwards compatible.

Reply:
sure, removed now/again